packages/fxa-settings/.rescriptsrc.js

```diff
@@ -8,8 +8,28 @@ const {
   componentsJestMapper,
 } = require('fxa-react/configs/rescripts');
 
+function loadServerConfig(config) {
+  const serverConfig = require('../fxa-content-server/server/lib/configuration');
```

Comment:
We need to be selective on what to send.  (For example, here's where the config for the content server frontend is defined: https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/server/lib/routes/get-index.js#L54.)

Reply:
Interesting. I think we were sending everything in the previous implementation?

Reply:
Would what's in your example above also be what we'd want in the settings frontend? Or some other subset?

Reply:
> 
> 
> Interesting. I think we were sending everything in the previous implementation?

Yikes

Reply:
> 
> 
> Would what's in your example above also be what we'd want in the settings frontend? Or some other subset?

I haven't been following the settings work that closely.  At the very least, we'd need something for the gql-api and sentry?

Reply:
Yeah, so we start with the content server config and merge it with settings-specific (including gql stuff and sentry) config [here](https://github.com/mozilla/fxa/blob/master/packages/fxa-settings/src/lib/config.ts).